src/app/shared/form-field/form-field.component.html

```diff
@@ -1,12 +1,18 @@
 <div [ngClass]="{ 'has-error' : !field[1].valid }">
 <label>{{ field[0].label }}</label>
 
-<span *ngFor="let validation of field[0].validations">
+<span id="{{field[0].name}}_error_block" *ngFor="let validation of field[0].validations">
   <em class="text-danger" *ngIf="!field[1].valid && field[1]['errors'][validation.type.toLowerCase()]"> {{ validation.message }}</em>
 </span>
 
 <div *ngIf="field[0].type == 'select'">
   <select [ngFormControl]="field[1]"
+    [id]="field[0].name"
```

Comment:
For now, We need the `id` just for tests purpose. 'll be need to look into it (i.e., directive thing).